deploy/src/erc_to_erc/home.js

```diff
@@ -152,6 +157,36 @@ async function initializeBridge({ validatorsBridge, bridge, erc677token, initial
   }
   nonce++
 
+  if (isRewardableBridge && BLOCK_REWARD_ADDRESS !== ZERO_ADDRESS && FOREIGN_TO_HOME_DECIMAL_SHIFT) {
+    console.log(`Initialize decimal shift with parameter :
+    FOREIGN_TO_HOME_DECIMAL_SHIFT: ${foreignToHomeDecimalShift}
+    `)
+    let initializeDecimalShiftData = await bridge.methods
+    .setDecimalShift(
```

Comment:
my understanding is that we can do this as part of `initialize` action, is it true?

Reply:
I try to add it to the `initialize` function but it leads to "stack too deep" error. That is why I split in two part as a technical workaround. Or maybe remove this feature for this bridge mode case and be able to remove the `setDecimalShift` function too.

Reply:
Ok. I see. So, it seems that it is good to start packing parameters in one variable. E.g. `_homeFee`, `_foreignFee` and `decimalShift` could be packed in one 32-bytes word.

Reply:
I try and succeed using params arrays for `rewardableInitialize()` function that allow to add `decimalShift` param at init step too : https://github.com/iExecBlockchainComputing/poa-bridge-contracts/commit/0a4adfd5361693319a3bccbcad87963a665049d2 . It allows also to remove the setter function. Is this Array parameters what you refer to or this is another packed parameters technic? Do I generalize packed parameters (for limits and fees parameters) to all initialize function or just where it is needed to avoid stack too deep ?

Reply:
Very well achievement! Yes, let's generalize the packed parameters inn all initialize functions

Reply:
OK, I  generalize packed parameters in this commit : https://github.com/poanetwork/poa-bridge-contracts/pull/268/commits/819634bc5fcbe450b68da9eeff9c050ba38adbda